### PR TITLE
Fix entity list refresh when entity deleted

### PR DESCRIPTION
**Pull request by dback2**
Fix the entity list not updating to remove any entity deleted via either keyboard Delete, lifetime elapsing, or external deleteEntity call.

Fixes: https://highfidelity.manuscript.com/f/cases/14127/

Test Plan:
- Enter Interface in desktop mode
- Open Create and create any entity
- Shift to the List tab and find the selected entity in the list
- Select the entity again to focus in world away from the Create window
- Press Delete key and verify the deleted entity is automatically removed from the list
- Create another entity and copy it's ID in the Properties tab
- Find the selected entity in the List tab
- Open Developer menu -> Console and enter the following line and press enter:
Entities.deleteEntity(XXX);
where XXX is the ID you copied from the Properties tab above
- Verify the deleted entity is automatically removed from the list
- Create another entity and set it's lifetime in the Properties tab to 30
- Find the selected entity in the List tab
- Verify that after the lifetime elapses and the entity is deleted that it is automatically removed from the list

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Android build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dandroid/11271/HighFidelity-Beta-PR13597-765dda4.apk

**Comment by hifi-gustavo** [comment]
Mac OS X build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11271/HighFidelity-Beta-PR13597-765dda4.dmg

**Comment by hifi-gustavo** [comment]
Mac OS X client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dmacos/11271/HighFidelity-Beta-Interface-PR13597-765dda4.dmg

**Comment by hifi-gustavo** [comment]
Windows build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11271/HighFidelity-Beta-PR13597-765dda4.exe

**Comment by hifi-gustavo** [comment]
Windows client-only build is available at: https://deployment.highfidelity.com/jobs/pr-build/label%3Dwindows/11271/HighFidelity-Beta-Interface-PR13597-765dda4.exe

File: scripts/system/edit.js [hunk 1]
```diff
@@ -1573,15 +1573,11 @@ function deleteSelectedEntities() {
                 Entities.deleteEntity(entityID);
             }
         }
-
+
```
Comment (thoys) [inline scripts/system/edit.js]: Don't you just love it when you IDE does this :)

**Comment by QliCknox** [comment]
Currently Testing

**Comment by QliCknox** [comment]
Pass @qliacoombs @Qliemillar 

Deleting entities works as expected

**Pull request merged**